CMakeLists.txt

```diff
@@ -443,6 +443,7 @@ if(BUILD_TESTS)
       raft_driver ${CMAKE_CURRENT_SOURCE_DIR}/src/consensus/aft/test/driver.cpp
     )
     target_link_libraries(raft_driver PRIVATE ccfcrypto.host)
+    use_client_mbedtls(raft_driver)
```

Comment:
?

Reply:
`use_*_mbedtls` should not be necessary when ccfcrypto.host/.enclave is linked in. I think we could remove those functions, but haven't actually tried to do so yet.

Reply:
Merge conflict gone wrong.